### DISCUSSION TITLE
TickTick sync

### DISCUSSION THREAD

**Discussion by iwex** [alainm23/planify#1627] [General]
Hello! It would be great to add TickTick (ticktick.com) sync support. It's very similar to todoist .

**Comment by RuthlessRuler** [comment]
Any updates on implementing this feature?

**Comment by Xanaxus** [comment]
I sent a mail to ticktick Support, let's see if they choose to help develop this feature,

@RuthlessRuler There is a deb (official) and flatpak packaging out there if u want to use it

**Comment by madx** [comment]
They do have an API which looks enough for a Planify integration https://developer.ticktick.com/api#/openapi

@alainm23 Are there any docs on how to implement a provider? Do we basically need to write a `TickTick.vala` file in `core/Services` or is there something more to it?

**Comment by crisqyxw** [comment]
I have to support this Feature Request as another voice! As a heavy user of Ticktick, I urgently hope it can integrate more deeply into my system.
